packages/components/tabs/src/TabPanel.tsx

```diff
@@ -17,9 +18,14 @@ function _TabPanel(
   }: TabPanelProps,
   ref: React.Ref<HTMLDivElement>,
 ): React.ReactElement {
-  return (
+  const { selectedTab } = useTabsContext();
+  const elementProps = {
+    'aria-labelledby': `${id}-control-tab`,
+  };
+  return selectedTab === id ? (
     <Box
       as="div"
+      {...elementProps}
```

Comment:
Isn't elementProps only `aria-labellledby`?

What do you think about passing it directly here instead?

Reply:
`Box` doesn't take it

Reply:
uhmmm It must be some problem in the typing

it's ok then, nevermind :)